### PR TITLE
Remove `both` value for `generate` key

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR removes the `both` value for the `generate` key. Specifying a list of values is more flexible for the future.

### CODE REVIEW

File: tests/test_rapids_dependency_file_generator.py [hunk 1]
```diff
@@ -80,3 +82,39 @@ def test_should_use_specific_entry():
...
+    result = get_file_types_to_generate(
+        [str(GeneratorTypes.REQUIREMENTS), str(GeneratorTypes.CONDA)]
+    )
+    assert result == [str(GeneratorTypes.REQUIREMENTS), str(GeneratorTypes.CONDA)]
+
+    with pytest.raises(Exception):
+        get_file_types_to_generate("asdas")
+
+    with pytest.raises(Exception):
+        get_file_types_to_generate(["asdas"])
```
Comment (bdice) [inline tests/test_rapids_dependency_file_generator.py:115]:
```suggestion
    with pytest.raises(ValueError):
        get_file_types_to_generate("invalid_value")

    with pytest.raises(ValueError):
        get_file_types_to_generate(["invalid_value"])
```

File: tests/test_rapids_dependency_file_generator.py [hunk 2]
```diff
@@ -80,3 +82,39 @@ def test_should_use_specific_entry():
...
+
+    with pytest.raises(Exception):
```
Comment (bdice) [inline tests/test_rapids_dependency_file_generator.py:117]:
```suggestion
    with pytest.raises(ValueError):
```

**Comment by ajschmidt8** [comment]
I also shortened the header in ac8288e.

**Review by bdice — APPROVED** [review]
LGTM, with minor suggestions. J'approuvé.

**Pull request merged**